pgmq-extension/README.md

```diff
@@ -185,36 +211,77 @@ SELECT pgmq.archive('my_queue', 2);
 (1 row)
 ```
 
+Or archive several messages in one operation using `msg_ids` (plural) parameter:
+
+First, send a batch of messages
+
+```sql
+
+SELECT pgmq.send_batch('my_queue', ARRAY['{"foo": "bar3"}','{"foo": "bar4"}','{"foo": "bar5"}']::jsonb[]);
```

Comment:
Do you want to use named parameters like the other examples?

Reply:
Ah yes, good call.

Reply:
https://github.com/tembo-io/pgmq/pull/289/commits/54609153261b96b27ade54ffa0b34f915e91c334